rs/sns/audit/src/lib.rs

```diff
@@ -199,17 +198,16 @@ async fn validate_neurons_fund_sns_swap_participation(
         .neurons_fund_reserves
         .unwrap()
         .neurons_fund_neuron_portions;
-    let mut initial_amounts_per_controller: BTreeMap<String, _> = BTreeMap::new();
+    let mut initial_amounts_per_controller: BTreeMap<_, _> = BTreeMap::new();
```

Comment:
Just drop the type. It is now conveying zero information.

Reply:
Thanks, good catch